kratos/utilities/model_part_utils.h

```diff
@@ -0,0 +1,61 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Máté Kelemen
+//
+
+// Project includes
+#include "includes/kratos_export_api.h"
+#include "includes/model_part.h"
+#include "includes/global_variables.h"
+
+namespace Kratos {
+
+
+/// @brief Class collecting a set of free-floating utility functions for querying and mutating @ref ModelPart s.
+class KRATOS_API(KRATOS_CORE) ModelPartUtils
```

Comment:
```suggestion
class ModelPartUtils
```

I don't think `KRATOS_API` is required for headers only classes.